examples/speaker_recognition/conf/SpeakerNet_ECAPA.yaml

```diff
@@ -0,0 +1,94 @@
+name: &name "SpeakerNet"
+sample_rate: &sample_rate 16000
```

Comment:
All of these values after name should be inside "model". Remove "&xyz" and use hydra interpolation